[FPADF] Announcement (#25)
FreeCAD/FPA: FPADF_Announcement.md

## <img src="images/freecad.svg" style="zoom:50%;" /> FreeCAD Project Association Development Fund

#### Announcement - FPA Development Fund

The FreeCAD Project Association is pleased to announce the creation of the FPA Development Fund (FPADF). The FPADF offers grants to individuals for contributions to FreeCAD.

> Initially, grants from the FPADF will be capped at US$1000 and grants will be awarded only for work on the FreeCAD code base.

Reviewer: ```suggestion
Grants are capped at US$1000 and will be awarded only for work on the FreeCAD code base.
```

Reviewer: Avoid casting the current announcment as suboptimal or inferior to a future goal.

The cap may be raised as the FPA gains experience in the awarding of grants and as available funds allow.

> It is expected that grants will eventually be made available to contributors working on documentation, packaging, and other aspects of FreeCAD.

Reviewer: ```suggestion

```

> Any contributor who wishes to be considered for a grant must submit a proposal in writing to the FPA. The contributor must state clearly how much money is requested and what the contributor intends to create/improve/fix. An estimated timeline with major milestones should be part of the grant application and clear completion criteria should be provided.

Reviewer: ```suggestion
Any contributor who wishes to be considered for a grant should submit a proposal in writing to the FPA. The proposal should state clearly how much money is requested and what the contributor intends to create/improve/fix. An estimated timeline with major milestones should be included along with clear completion criteria.
```

Proposals will be considered by the FPA general body and approved by vote.

Grant applications may be forwarded to any member of the FPA for consideration.

> 

Reviewer: ```suggestion
The FreeCAD Project Association would like to expand the number and type of grants given.  Donations made to the FPA and designated DEVELOPMENT FUND will be used for this purpose. 
```
